src/test/java/FormAuthenticationTest.java

```diff
@@ -0,0 +1,54 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class FormAuthenticationTest extends BasicSetupTest {
+	/**
+	 * 5.
+	 * This test navigates to the "Form Authentication" page, enters valid credentials, and verifies successful login.
+	 * It then logs out of the system and confirms that the user is successfully logged out, with the login button becoming available again.
+	 */
+	@Test
+	public void formAuthenticationTest() {
+		browser.get("https://the-internet.herokuapp.com/login");
+
+		WebElement header = browser.findElement(By.xpath("//h2['Login Page']"));
+		Assert.assertEquals(header.getText(), "Login Page", "The title of the page should be 'Login Page'.");
+
+		WebElement usernameField = browser.findElement(By.id("username"));
+		WebElement passwordField = browser.findElement(By.id("password"));
+		Assert.assertTrue(usernameField.isDisplayed(), "The login field must be displayed.");
+		Assert.assertTrue(passwordField.isDisplayed(), "The password field must be displayed.");
+
+		usernameField.sendKeys("tomsmith");
```

Comment:
String values can change with a time - so we better extract them to variables and keep them on top of the test method (ideally this is managed in a different manner but the extract+keep on the top would be enough for now)

Hardcoded values for method params should be avoided when possible